modules/roles_profiles/manifests/profiles/nxlog.pp

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class roles_profiles::profiles::nxlog {
+
+    case $::operatingsystem {
+        'Windows': {
+            require  win_packages::nxlog
+
+            $programfilesx86 = $facts['programfilesx86']
+
+            file { "${programfilesx86}\\nxlog\\cert\\papertrail-bundle.pem":
```

Comment:
Is this file really required if we are sending logs to log aggregators via syslog?  If this file is really needed, it should at least be updated since it is 5 years old

Reply:
I think this file is needed on the cloud side of things.

Reply:
It is needed for AWS: https://github.com/mozilla-releng/OpenCloudConfig/blob/4978a75a872847deadac1f72eec8ad3bdbcf15a4/userdata/Configuration/nxlog/win10.conf#L130

Just a note the nxlog pieces are going to continue to grow and become more complex as as we add  support for additional OSes and location type support.

Reply:
I know that recently changed on the cloud side but I still think logs should be sent to aggregators and not directly to papertrail.  We need to fix the issue causing logs to be broken when sent to the log aggregators in the first place.

Reply:
I think that is outside of the scope of this. 

To the best of my understanding cloud instances always went directly to papertrail. The recent proposed change was to send hardware logs to papertrail directly. To change this we need to bring Rob into the conversation. Also note that the AWS instance report to TaskCluster papertrail. Which also something we may want to change in the future. 

For now I have moved it into a conditional clause based on location.